pkg/allowdenylist/allowdenylist_test.go

```diff
@@ -124,22 +140,38 @@ func TestInclude(t *testing.T) {
 			t.Fatal("expected New() to not fail")
 		}
 
-		denylist.Exclude([]string{"kube_node_.*_cores", "kube_pod_.*_bytes"})
+		denylist.Exclude([]string{"kube_(?=node.*cores|pod.*bytes)"})
```

Comment:
Is it compatible with "denylist.Exclude([]string{"kube_node_.*_cores", "kube_pod_.*_bytes"})"?

Lookaround feature is more complicated. I will prefer using the legacy one unless performance is much better.

Reply:
It is. The library being introduced here is a superset of go's regexp one, and will only incur expensive perf operations for exponential matches, that go's library doesn't support since it has to guarantee linear-time operations.

Reply:
Also these operations are capped at a minute.